modules/grandpa/src/weights.rs

```diff
@@ -55,9 +56,9 @@ pub trait WeightInfo {
 pub struct MillauWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for MillauWeight<T> {
 	fn submit_finality_proof(p: u32, v: u32) -> Weight {
-		(115_651_000 as Weight)
-			.saturating_add((61_465_000 as Weight).saturating_mul(p as Weight))
-			.saturating_add((3_438_000 as Weight).saturating_mul(v as Weight))
+		(55_070_000 as Weight)
```

Comment:
The difference between the old value and the new one seems quite big. Here and in other places as well. Is this normal / expected ?

Reply:
Yeah. I've also made a comment regarding that in the PR description. Outline: normally (for non-test runtimes) we are running benchmarks (using benchmarking bot) on the dedicated reference machine. But here (for testnets) we don't need such precision, so I've been using my own laptop for that && recently I've upgraded it :) Additionally, the messages pallet configuration has changed significantly, so it is normal that we see drop there